pkg/providers/vsphere/contentlibrary/content_library_utils_test.go

```diff
@@ -149,6 +169,13 @@ var _ = Describe("UpdateVmiWithOvfEnvelope", func() {
 		Expect(image.Status.VMwareSystemProperties[0].Value).Should(Equal(versionVal))
 
 		Expect(conditions.Has(image, vmopv1.VirtualMachineImageV1Alpha1CompatibleCondition)).To(BeFalse())
+
+		Expect(len(image.Status.DiskInfo)).To(Equal(2))
```

Comment:
There is a `HaveLen` matcher that gives a more details when the length doesn't match

`Expect(image.Status.DiskInfo).To(HaveLen(2))`